src/main/java/io/vertx/core/http/impl/ClientConnection.java

```diff
@@ -414,7 +414,9 @@ protected synchronized void handleClosed() {
 
     // Connection was closed - call exception handlers for any requests in the pipeline or one being currently written
```

Comment:
Shouldn't the comment be updated as well ? The request being currently written will not have its exception handler invoked